brokers/vscode/broker_test.go

```diff
@@ -0,0 +1,220 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package vscode
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/eclipse/che-plugin-broker/model"
+)
+
+func Test_findAssetURL(t *testing.T) {
+	type args struct {
+	}
+	tests := []struct {
+		name     string
+		response []byte
+		want     string
```

Comment:
I think it would be a bit simpler to have `want string` and `wantError string` instead of three fields.